nav2_route/src/plugins/graph_file_loaders/geojson_graph_file_loader.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2023 Joshua Wallace
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <fstream>
+
+#include "nav2_route/plugins/graph_file_loaders/geojson_graph_file_loader.hpp"
+
+namespace nav2_route
+{
+
+bool GeoJsonGraphFileLoader::loadGraphFromFile(
+  Graph & graph, GraphToIDMap & graph_to_id_map, std::string filepath)
+{
+  std::ifstream graph_file(filepath);
+
+  Json json;
+  graph_file >> json;
+
+  auto features = json.at("features");
```

Comment:
Is features a universal thing or do we need to make that a parameter to search or and/or grab this from the file's root itself?

Reply:
This is universal for geojson